examples/advanced_operations/add_responsive_search_ad_full.py

```diff
@@ -405,39 +515,28 @@ def add_geo_targeting(client, customer_id, campaign_resource_name):
         [_GEO_LOCATION_1, _GEO_LOCATION_2, _GEO_LOCATION_3]
     )
 
-    results = geo_target_constant_service.suggest_geo_target_constants(
-        gtc_request
-    )
+    results = geo_target_constant_service.suggest_geo_target_constants(gtc_request)
 
     operations = []
     for suggestion in results.geo_target_constant_suggestions:
-        geo_target_constant = suggestion.geo_target_constant
         print(
-            f"{geo_target_constant.resource_name} "
-            f"({geo_target_constant.name}, "
-            f"{geo_target_constant.country_code}, "
-            f"{geo_target_constant.target_type}, "
-            f"{geo_target_constant.status.name}) "
+            f"geo_target_constant: {suggestion.geo_target_constant} "
```

Comment:
what does this print? Should it be the resource name or the name? Or is it intentionally the full object?